GitHub-resources/DIME-GitHub-Templates/gitignore_template.txt

```diff
@@ -48,6 +47,10 @@
 !/**/*.R
 !/**/*.Rmd
 !/**/*.Rproj
+# Still ignore shiny app meta files
+**/rsconnect/**/*.dcf
+# Still ignore user file for R projects
+.Rproj.user
```

Comment:
@luizaandrade does this look right to you? These should both remained ignored, right?

Reply:
yes, that's right! sorry for the confusion